libevmasm/Instruction.cpp

```diff
@@ -79,7 +79,7 @@ std::map<std::string, Instruction> const dev::eth::c_instructions =
 	{ "COINBASE", Instruction::COINBASE },
 	{ "TIMESTAMP", Instruction::TIMESTAMP },
 	{ "NUMBER", Instruction::NUMBER },
-	{ "DIFFICULTY", Instruction::DIFFICULTY },
+	{ "PREVRANDAO", Instruction::PREVRANDAO },
```

Comment:
{ "PREVRANDAO", Instruction::PREVRANDAO } instead for DIFFICULTY support?

Reply:
No, we need to support both